src/aapns/connection.py

```diff
@@ -0,0 +1,440 @@
+from __future__ import annotations
+
+import json
+from asyncio import (
+    CancelledError,
+    Event,
+    Future,
+    TimeoutError,
+    create_task,
+    open_connection,
+    wait_for,
```

Comment:
missing `Task` import for the dataclass. And while more verbose, I think this file should import `asyncio` (rather than things from there), mostly to distinguish `asyncio.Event` from `h2.event.Event` (my first reading of the file I got those two confused).

Reply:
So far I was trying to:
* for stdlib, use `from mod import Foo`
* for 2rd party use `import mod.submod`

However, I wonder, if, perhaps, specifically for annotated types (e.g. public function args), I should use fully qualified name.

I mean, while knows where `def conn(arg: SSLContext)` is really coming from?
(sure mypy does, but it's hard for a human reader)